### PR TITLE
Review status inferrement

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**
Feat
Can pass status in queryParams to get reviews belonging to that status. Values are `active`, `pending`, or `inactive`

### CODE REVIEW

File: api/app/utils/review.inference.js
Comment (Extheoisah) [inline api/app/utils/review.inference.js:32]: if we add this condition, how do we ascertain that an inactive transcript should be re-queued (given that it has expired but not submitted)? Is this accounted for somewhere else in the code base?

**Review by Extheoisah — COMMENTED** [review]
Looks good! Just one comment.

File: api/app/utils/review.inference.js
Comment (Emmanuel-Develops) [reply api/app/utils/review.inference.js:32]: These conditions are for getters. e.g. Get all inactive from a `/reviews` API call. i.e find all reviews that this condition satisfies

Requeue is done from either of 2 processes:
- CRON job: A cron is created once a transcript is claimed (review is created) we pass the reviewId to the CRON and it is scheduled to run exactly 24 hours after it has been created. It doesn't use this condition as we don't do a findAll, rather get that review from the reviewId then if it has any timeStamp (submittedAt, mergedAt, archivedAt) don't requeue the transcript associated with that review. 
- Webhook: If the PR opened against a review is closed by the repo maintainer (not merged) find the review associated with that PR_URL and requeue the associated transcript

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**
